src/test/groovy/net/fabricmc/loom/test/unit/layeredmappings/ParchmentMappingLayerTest.groovy

```diff
@@ -38,16 +38,18 @@ class ParchmentMappingLayerTest extends LayeredMappingsSpecification {
             def mappings = getLayeredMappings(
                     new IntermediaryMappingsSpec(),
                     new MojangMappingsSpec(),
-                    new ParchmentMappingsSpec(PARCHMENT_NOTATION, false)
+                    new ParchmentMappingsSpec(PARCHMENT_NOTATION, true)
             )
             def tiny = getTiny(mappings)
+            def reorderedMappings = reorder(mappings)
         then:
             mappings.srcNamespace == "named"
             mappings.dstNamespaces == ["intermediary", "official"]
             mappings.classes.size() == 5747
             mappings.classes[0].srcName.hashCode() == -1112444138 // MojMap name, just check the hash
             mappings.classes[0].getDstName(0) == "net/minecraft/class_2573"
-            mappings.classes[0].methods[0].args[0].srcName == "pStack"
+            mappings.classes[0].methods[0].args[0].srcName == "stack"
```

Comment:
Please check the hashcode of this. See 2 lines above.

Reply:
Is there a difference? I will do it anyway tho

Reply:
Oh my bad it was wrong before anyway. Prob doesn’t matter. Thanks